### ISSUE TITLE
Crash when crop PNG file

### ISSUE THREAD

**Issue by ykying** [IGRSoft/IGRPhotoTweaks#35]
CGImage+IGRPhonoTweakExtension.swift
CGContext is null when cropping PNG file

**Comment by gholias** [comment]
The same problem is happening to me

**Comment by gholias** [comment]
This is the error message
```
[Unknown process name] CGBitmapContextCreate: unsupported parameter combination: set CGBITMAP_CONTEXT_LOG_ERRORS environmental variable to see the details
```

**Comment by joshgrenon** [comment]
Any progress on this error? Getting it also.

**Comment by iKorich** [comment]
@joshgrenon Thank you for the fix 731e377874ee546d4fd7e93bd61988d0e882e792

**Comment by owlstack** [comment]
@joshgrenon thank you for the fix! I was getting the same issue for pngs too!
